localops/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "localops"
```

Comment:
If this is not private, should be `iroh-localops` for now

Reply:
can we for now just put this functionality into `iroh-util`?

Reply:
I'd heavily prefer `iroh-localops`, even just for the amount of conditional compilation that needs to go into this crate. Most of this work is about vetting dependencies for cross-platform compatibility

Reply:
how does that change when putting it in utils? we already have a lot of crates :/